src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawWriter.cs

```diff
@@ -312,6 +321,12 @@ internal virtual void WriteEndBase64()
             _base64Encoder.Flush();
         }
 
+        internal virtual void WriteSpan(ReadOnlySpan<char> value)
+        {
+            //fallback if not overriden
+            WriteString(new string(value));
```

Comment:
```suggestion
            WriteString(value.ToString());
```
(it's the same, but IMO a bit more readable)